### PR TITLE
 made an error message more approachable in rust/library/core/src/str/mod.rs:116

### PR DESCRIPTION
**Pull request by gorentbarak**

### CODE REVIEW

**Comment by gorentbarak** [comment]
I also answered the question `sup?` in the code.

**Comment by slanterns** [comment]
Please remove the merge commit first (see https://rustc-dev-guide.rust-lang.org/git.html#keeping-things-up-to-date.) Also please avoid touching unrelated things in your PR just for fun.

File: src/tools/rustfmt/tests/source/extern.rs [hunk 1]
```diff
@@ -47,6 +47,7 @@ extern "Rust" { static ext:  u32;
 
 extern "C" {
             fn syscall(number: libc::c_long /* comment 1 */, /* comm 2 */ ... /* sup? */) -> libc::c_long;
+	// im doing great! thanks for asking :).
```
Comment (ytmimi) [inline src/tools/rustfmt/tests/source/extern.rs:50]: Was this change necessary? If not can you please revert it.

**Review by ytmimi — CHANGES_REQUESTED** [review]
rustfmt changes should only be made in rust-lang/rust when absolutely necessary

**Comment by TaKO8Ki** [comment]
Could you fix CI errors? Please refer to this documentation for running tests locally.

https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html#common-x-commands

File: src/tools/rustfmt/tests/source/extern.rs (hunk 1 above)
Comment (ytmimi) [reply src/tools/rustfmt/tests/source/extern.rs]: Thank you for removing the comment.

**Comment by JohnCSimon** [comment]
@gorentbarak 

ping from triage - can you post your status on this PR? This PR has not received an update in a few months.


FYI: when a PR is ready for review, send a message containing 
`@rustbot ready` to switch to `S-waiting-on-review` so the PR is in the reviewer's backlog.

Or if you're not going to continue, please close it. Thank you!

**Comment by Dylan-DPC** [comment]
Closing this as inactive. Feel free to reöpen this pr or create a new pr if you get the time to work on this. Thanks

**Pull request closed without merging**